src/components/AssetDocumentsPanel/stories/basic.story.mdx

```diff
@@ -0,0 +1,89 @@
+import { Meta, Story, Preview, Props, Source } from '@storybook/addon-docs/blocks';
+import { AssetDocumentsPanel } from '../AssetDocumentsPanel';
+import { decorators } from './helper';
+
+<Meta title="Assets|AssetDocumentsPanel" decorators={decorators} />
+
+ # Asset Documents Panel
+
+This component loads list of documents related to the asset and presents it in the list of categories.
+Each item in the list can be expanded showing list of files.
+The component is used as one of the panes in `AssetMeta` component.
+
+**NOTE:** The component should have `ClientSDKProvider` as a parent component in react component tree.
+
+## Basic usage:
+
+Don't forget to import:
```

Comment:
"Don't forget" is something that you say after you gave some instructions before
but doesn't really sound good on the beginning.
can be replaced with just `Imports:`